### ISSUE TITLE
Problem: Processing Configuration Docs are out of date

### ISSUE THREAD

**Issue by ross-spencer** [archivematica/Issues#100]
**Version of the documentation**

1.7.1

**Page (and section, if applicable) where the issue occurs**

https://www.archivematica.org/en/docs/archivematica-1.7/admin-manual/installation-setup/customization/dashboard-config/#process-config

**Description of the issue**

There are a few changes that can be made to this page, a few suggestions:

* Document the ability from 1.8 onwards to download a processing configuration:

![image](https://user-images.githubusercontent.com/1880412/44228566-36e6ee80-a15b-11e8-85b7-d589c30360e8.png)

* Remove the detail about each specific choice as it can quickly become out of date, and is no longer up-to-date with `stable/1.7`

* Maybe replace the above with a few highlights (it might be hard to pinpoint important options), but examples might be, where to store the AIP or DIP. We might emphasize the impact of choosing `None` vs. other options.

* Include only a snippet of the information about the internal structure of a processing configuration. I don't believe that this impacts the end-user at this point in time. 

* Provide a little more detail describing the possible transfer structures for including a custom configuration, e.g.:
```
ross-spencer@artefactual:~/git/artefactual-labs/am/src/archivematica-sampledata/custom-transfers$ tree processing-config-test-1
processing-config-test-1
├── objects
│   └── test-file.txt
└── processingMCP.xml

1 directory, 2 files
```
or:
```
ross-spencer@artefactual:~/git/artefactual-labs/am/src/archivematica-sampledata/custom-transfers$ tree processing-config-test-2
processing-config-test-2
├── processingMCP.xml
└── test-file.txt

0 directories, 2 files
```
**NB.** If a user downloads a config with `<name>.xml` it needs to be renamed (case-sensitive) `processingMCP.xml` in the transfer. 

* Update the config image where the old-style check-boxes still appear: 

![image](https://user-images.githubusercontent.com/1880412/44229086-9a255080-a15c-11e8-834f-bbcbc03a993b.png)

**Comment by sallain** [comment]
@ross-spencer - I didn't see this ticket before I made extensive changes to the dashboard config page to address #170. Note that there are now two dashboard config pages - one for [admins](https://www.archivematica.org/en/docs/archivematica-1.7/admin-manual/installation-setup/customization/dashboard-config/#processing-configuration) and one for [users](https://www.archivematica.org/en/docs/archivematica-1.7/user-manual/administer/dashboard-admin/#dashboard-processing).

I like some of your suggestions from above, so I'll try to capture them here in a checklist:

- [ ] Document the ability from 1.8 onwards to download a processing configuration
- [ ] Include only a snippet of the information about the internal structure of a processing configuration. I don't believe that this impacts the end-user at this point in time - this is now in the admin-manual specific page, so a standard user would not see it
- [ ] Provide a little more detail describing the possible transfer structures for including a custom configuration - I think I've clarified this in the docs changes related to #170, but a tree wouldn't be a bad addition
- [ ] Update the config image where the old-style check-boxes still appear

re: _Remove the detail about each specific choice as it can quickly become out of date, and is no longer up-to-date with stable/1.7_, I think I'd prefer to just try keeping this up to date - there is no other place in the documentation where a lot of these choices are described, and figuring out what all of this means is pretty big task for users.
